docs/supported_languages_and_lockfiles.md

```diff
@@ -81,6 +81,14 @@ After the dependency resolution, the OSV database is queried for the vulnerabili
 {: .note }
 Test dependencies are not supported yet in the computed dependency graph for Maven pom.xml.
 
+### Data source
+
+By default, we use the [deps.dev API](https://docs.deps.dev/api/v3/) to find version and dependency information of packages during transitive scanning.
+
+If instead you'd like to fetch data from [Maven Central](https://repo.maven.apache.org/maven2/), you can use the `--experimental-resolution-data-source=native` flag.
+
+If your project uses mirrored or private registries, in addition to setting `--experimental-resolution-data-source=native`, you will need to use the `--experimental-maven-registry=<full-registry-url>` flag to specify the registry (e.g. `--experimental-maven-registry=https://repo.maven.apache.org/maven2/`).
```

Comment:
Unrelated to documentation, but should we name the flag `--resolution-registry` to be more generic? Or is it very specific to Maven?

Reply:
I have thought of this considering we will soon support transitive scanning for Python requirements.txt - ideally I would like to have something `--resolution-registry=Maven:[URL],PyPI:[URL]`, but I don't have a very clear picture about how does Python private registry work so I don't want to put too much effort on finalizing this flag now, and this is also why I mark this flag as experimental. Also for the ecosystem string to be used in the flag, this should follow the spec that Gareth is working on which is also not finallized.

Reply:
SGTM